packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -527,7 +611,24 @@ const SideNav = ({
                             isDisabled={isDisabled}
                           >
                             <SideNavListContainer id={`${id}-list`} role="list">
-                              {children}
+                              {isSortable ? (
+                                <SortableList
+                                  parentId={item.id}
```

Comment:
Is this for drag 'n drop? Just takes an `id` prop?

Reply:
yes, this is to track the parent accordion id in order to update the items after the drop